YoCode/FileImport.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace YoCode
+{
+    // To import:
+    // UnitConverter.cs from junior-test/UnitConverterWebApp
+    // Program.cs from junior-test/UnitConverterWebApp
+    // Startup.cs from junior-test/UnitConverterWebApp
+    // HomeController.cs from junior-test/UnitConverterWebApp/Controllers
+
+    class FileImport
+    {
+        public string ORIGINAL_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\junior-test";
+        public string MODIFIED_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\original-test";
+
+        List<String> SearchPatterns = new List<string>{ "*.cs", "*.cshtml", "*.js" };
+        
+        //Will return a list of all files from a directory
+        public List<string> GetAllFilesInDirectory(String PATH)
+        {
+            List<string> files = new List<string>();
+            DirectoryInfo di = new DirectoryInfo(PATH);
+            FileInfo[] fileinfo = di.GetFiles("*", SearchOption.AllDirectories);
+
+            AddFileInfoToList(files, fileinfo);
+
+            return files;
+
+        }
+
+        //Will return a list of files from a directory given a pattern
+        public List<string> GetFilesInDirectory(String PATH, String pattern)
+        {
+            List<string> files = new List<string>();
+            DirectoryInfo di = new DirectoryInfo(PATH);
+            FileInfo[] fileinfo = di.GetFiles(pattern, SearchOption.AllDirectories);
+
+            AddFileInfoToList(files, fileinfo);
+
+            return files;
+
+        }
+
+        //Will return a list of files from a directory given a list of patterns
+        public List<string> GetFilesInDirectory(String PATH, List<String> searchPatterns){
+
+            DirectoryInfo di = new DirectoryInfo(PATH);
+            int len = searchPatterns.Count;
+            List<String> files = new List<String>();
+
+            for (int i = 0; i < len; i++)
```

Comment:
I think this could be `foreach(var pattern in searchPatterns)` because you don't need the index, you just need the item from the list to pass to `GetFiles`.